src/django_components/component.py

```diff
@@ -275,6 +261,10 @@ def __new__(mcs, name: Any, bases: Tuple, attrs: Dict) -> Any:
 
     # This runs when a Component class is being deleted
     def __del__(cls) -> None:
+        # Skip if `extensions` was deleted before this registry
+        if not extensions:
```

Comment:
This, together with [this change in `component.registry.py`](https://github.com/django-components/django-components/pull/1138/files#diff-a5dcf86b1b1bd690092e8ed60318835a1ce9ed2c62befcc05d43dfd4459393b6R263) fixes https://github.com/django-components/django-components/issues/1133